lib/components/user/account-page.js

```diff
@@ -15,18 +16,25 @@ import SubNav from './sub-nav'
  * wrap any user account page (e.g., SavedTripList or account settings).
  */
 class AccountPage extends Component {
-  componentDidMount () {
-    const { loggedInUser, routeTo } = this.props
+  /**
+   * If a user signed up in Auth0 and did not complete the New Account wizard
+   * (and they are not on or have not just left the Terms and Conditions page),
+   * make the user finish set up their accounts first.
+   * monitoredTrips should not be null otherwise.
+   * NOTE: This check applies to any route that makes use of this component.
+   */
+  _checkAccountCreated = () => {
```

Comment:
I'd rather not bury this logic/routing behavior in a method. Is there a reason for this?

Reply:
I was trying to address the case when we'd have to check in `componentDidUpdate`, e.g. when someone types in the URL for another account page before they agree on the terms, but I didn't find a good way to redirect without flicker...